### PR TITLE
test: expand fuzzing across SEC codes

### PR DESCRIPTION
**Pull request by adamdecaf**
Slowly adding more to the "fully automated fuzzing" goal of mine.

Issue: https://github.com/moov-io/ach/issues/311

### CODE REVIEW

File: test/fuzz-reader/makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 
 fuzz:
-	go-fuzz-build github.com/moov-io/ach/test/fuzz-reader
-	go-fuzz -bin=./fuzzreader-fuzz.zip -workdir=$(shell pwd)
+	GO111MODULE=off go-fuzz-build github.com/moov-io/ach/test/fuzz-reader
+	GO111MODULE=off go-fuzz -bin=./fuzzreader-fuzz.zip -workdir=$(shell pwd)
 
 install:
-	go get -u github.com/dvyukov/go-fuzz/...
+	GO111MODULE=off go get -u github.com/dvyukov/go-fuzz/...
```
Comment (adamdecaf) [inline test/fuzz-reader/makefile:8]: `go-fuzz` isn't upgraded to Go 1.11 yet, so we have to disable Go Modules.

**Review by wadearnold — COMMENTED** [review]
Do we need more files for a larger corpus?

**Comment by adamdecaf** [comment]
@wadearnold A wide variety of files would be good, yea.

**Comment by wadearnold** [comment]
@adamdecaf merge this whenever you feel it should be added to master.

**Pull request merged**